### ISSUE TITLE
Method: "Decorative Images" Section of the WCAG 3  vs WCAG 2.x "Pure Decoration"

### ISSUE THREAD

**Issue by lauracarlson** [w3c/wcag3#238]
In WCAG 2.x [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration) is "serving only an aesthetic purpose, providing no information, and having no functionality". I don't know why we would [change it in WCAG 3](https://www.w3.org/WAI/GL/WCAG3/2020/methods/decorative-images/).  

What research is this change based upon?

If the purpose of an image is purely aesthetic or if it is redundant and does not relate to the content of the document, then use a null attribute alt="". If an image has a purpose then the user should be aware that it is there and what its intended purpose is.

A null alt attribute will be completely ignored by assistive technologies and will also prevent a person who is blind from knowing about and sharing the image. The author would be deciding that informing a user that the image on the page isn't important or worthwhile for them to know about, and so by using alt="" that fact is completely hidden from them.

To me, ultimately, it is about treating people with visual disabilities as users who access content differently, but equally. I want to treat them fairly and on an even footing: "deciding" that they likely don't want to know that an image is on the page is a serious decision to be taken - would you for example not include the image, and instead simply provide a link that read "click here to see a picture of a "Someone smiling and eating a salad"? I think not. Please reconsider this example.

Let's try to do our best to treat people with visual disabilities with that same kind of respect - afford them a textual glance of the image that all sighted users will make when the page first loads (and avoid going into too much detail, because few sighted users are going to study the image thoroughly).

So seriously let's consider if an image is purely decorative. If it isn't, authors should try to concisely explain what the image is in the context of the document's purpose without overly duplicating the text in which the image is embedded.

As [Eric Bailey, wrote](https://www.smashingmagazine.com/2021/06/img-alt-attribute-alternate-description-decorative/)

> "In modern web design and development, displaying an image is a highly intentional act. Alternate descriptions allow us to explain the content of the image, and in doing so, communicate **why it is worth including**.
> 
> Just because an image displays something fanciful doesn't mean it isn't worth describing. Announcing its presence ensures that anyone, regardless of ability or circumstance, can fully understand your digital experience." - @ericwbailey  

Thank you,
Laura
